docs/sdk/api/sdkAuth.md

```diff
@@ -24,10 +24,11 @@ Creates an auth client to handle authorization against the commercetools platfor
 #### Named arguments (options)
 
 1.  `host` _(String)_: the host of the OAuth API service
-2.  `projectKey` _(String)_: the key of the project to assign the default scope to
-3.  `credentials` _(Object)_: the client credentials for authentication (`clientId`, `clientSecret`)
-4.  `scopes` _(Array)_: a list of [scopes](https://docs.commercetools.com/http-api-authorization.html#scopes) (default `manage_project:{projectKey}`) to assign to the OAuth token
-5.  `fetch` _(Function)_: A `fetch` implementation which can be e.g. `node-fetch` or `unfetch` but also the native browser `fetch` function. Only needs be be passed if not globally available (e.g. through `isomorphic-fetch`)
+2.  `projectKey` _(String)_: the key of the project to assign the default scope to (optional).
+3.  `disableRefreshToken` _(boolean)_: whether the API should generate a refresh token
```

Comment:
Out of curiosity, how did you find out about this parameter? I cannot find it in the platform's documentation 😮

Reply:
I was walking through some already created projects where I found it.
If you set `disableRefreshToken` to `true`, it will add `&refresh_token=false` to form params and then API won't generate refresh token.